src/components/filter/filter.jsx

```diff
@@ -0,0 +1,195 @@
+          </label>
+        </li>
+        <li className="list-style-none">
+          <label className="fs-s form-label">
+            <input
+              type="radio"
+              className="form-checkbox"
+              name="sort"
+              value="low-to-high"
+              checked={state.filters["sortBy"] === "LOW_TO_HIGH"}
+              onChange={() =>
+                dispatch({ type: "SORT_BY", payload: "LOW_TO_HIGH" })
+              }
+            />
+            Price Low to High
+          </label>
+        </li>
+        <div className="divider-line"></div>
+        <li className="list-style-none filter-title">
+          <h3 className="fs-s">CATEGORIES</h3>
+        </li>
+        <li className="list-style-none">
+          <label className="form-label fs-s">
+            <input
+              type="checkbox"
+              className="form-checkbox"
+              checked={state.filters.categories.includes("FPS")}
+              onChange={() => dispatch({ type: "CATEGORIES", payload: "FPS" })}
+            />
+            FPS
+          </label>
+        </li>
+        <li className="list-style-none">
+          <label className="form-label fs-s">
+            <input
+              type="checkbox"
+              className="form-checkbox"
+              checked={state.filters.categories.includes("RACING")}
+              onChange={() =>
+                dispatch({ type: "CATEGORIES", payload: "RACING" })
+              }
+            />
+            Racing
+          </label>
+        </li>
+        <li className="list-style-none">
+          <label className="form-label fs-s">
+            <input
+              type="checkbox"
+              className="form-checkbox"
+              checked={state.filters.categories.includes("openWorld")}
+              onChange={() =>
+                dispatch({ type: "CATEGORIES", payload: "openWorld" })
+              }
+            />
+            Open World
+          </label>
+        </li>
+        <li className="list-style-none">
+          <label className="form-label fs-s">
+            <input
+              type="checkbox"
+              className="form-checkbox"
+              checked={state.filters.categories.includes("FIFA")}
+              onChange={() => dispatch({ type: "CATEGORIES", payload: "FIFA" })}
+            />
+            FIFA
+          </label>
+        </li>
+        <div className="divider-line"></div>
+        <li className="list-style-none filter-title">
+          <h3 className="fs-s">RATINGS</h3>
+        </li>
+        <li className="list-style-none">
+          <label className="form-label fs-s">
+            <input
+              type="radio"
+              name="rating"
+              className="form-checkbox"
+              onChange={() => dispatch({ type: "RATINGS", payload: 1 })}
+            />
+            1 <BiStarFill className="fs-s" /> and Above
+          </label>
+        </li>
+        <li className="list-style-none">
```

Comment:
I think, you can use Array here that will store the data for each radio button in an object. And then you can loop over that Array, and this `<li>` structure there.

Here all `<li>` has same structure, so you can do this.